rerun_py/src/dataframe.rs

```diff
@@ -469,6 +489,23 @@ pub struct PySchema {
 /// [`RecordingView.schema()`][rerun.dataframe.RecordingView.schema].
 #[pymethods]
 impl PySchema {
+    fn __str__(&self) -> String {
+        self.component_columns()
+            .iter()
+            .filter(|field| {
+                !field.entity_path().contains("link") && !field.component().contains("Indicator")
+            })
```

Comment:
```suggestion
            .filter(|column| !column.is_indicator)
```

I don't think we should filter out entity paths that contain the word `link`, even if the output contains a full URDF tree with lots of `**/link*/**` entities, these are valid columns that can contain valid data.

Reply:
I like the `pretty_print` idea. However, I don't think we should filter anything out (besides indicators) in both `__str__` and `__repr__`.

The problem with filtering out `"link"` at this level, is that there are cases where you do not want to filter this out. This will 100% lead to data being filtered out, which you don't want being filtered out.

I think we should definitely hide them in Paul's example, but not in the library code.

For Paul's filtering code, also don't think we should be checking for the string "Indicator" in the component name, we should maybe add a temporary getter for the `is_indicator` property. But there's #10315 and Indicators will be dropped (hopefully) soon anyways.

Reply:
I agree with @oxkitsune. I filtered out the links in my example because there are tons of them and it's kind of noise for our purposes. But in general, others may need/want to see this column information.

I would like to see something where we can filter the schema itself, like `print([col for col in dataset.schema() if 'link' not in col.name.lower()])`, and these columns pretty-print in the same fashion

Reply:
oh I missed the 2nd half of your example above. Nice! Sounds like we are on the same page then. Disregard